### DISCUSSION TITLE
New to GitHub

### DISCUSSION THREAD

**Discussion by aruimveld** [FuseFairy/DiscordBot-EdgeGPT#157] [Q&A]
Hello! I'm new to GitHub and really wanted to know how to get this put on my Discord Server. From what I can gather, I have to actually run a Python program to utilize this? If I'm wrong, where would I go to simply add the bot to my server?

Thank you. Sorry for my ignorance.

**Comment by FuseFairy** [comment]
Yes, you need to run the code yourself and add the bot to Discord.
